pages/documentation.js

```diff
@@ -0,0 +1,236 @@
+import React from 'react'
+import {flowRight} from 'lodash'
+
+import attachI18n from '../components/hoc/attach-i18n'
+import attachSession from '../components/hoc/attach-session'
+
+import Page from '../components/page'
+import Meta from '../components/meta'
+import Content from '../components/content'
+import Container from '../components/container'
+
+const DocumentationPage = () => (
+  <Page>
+    <Meta title='Documentation' />
+
+    <Content>
+      <Container>
```

Comment:
One container for the whole page is enough, you don’t need one for each section.

If you want multiple sections, use `<section>`.